README.md

```diff
@@ -111,6 +111,7 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [Feathers](http://feathersjs.com/) - is a real-time, micro-service web framework that gives you control over your data via RESTful resources, sockets and flexible plug-ins.
 * [Expressa](https://github.com/thomas4019/expressa) - Express middleware for creating APIs from JSON schemas with a simple admin editor and permissions model.
 * [rest-hapi](https://github.com/JKHeadley/rest-hapi) - Generate RESTful API based on mongoose models that supports relational data.
+* [oas-tools](https://github.com/isa-group/oas-tools) - NodeJS module to manage RESTfull APIs defined with OpenAPI 3.0 Specs over express servers.
```

Comment:
I don't understand the description. What's "managing an API"? Is it generation ? Stats? Please be more specific. 

Also, I suggest that you add swagger-tools, too.

Reply:
In oas-tools, 'managing' means automatic routing and spec validation. Moreover, a series of security features are also available, and a web portal for testing is created using Swagger UI.
Let me know if you need more information or if I should expand the description in the readme file.
I added swagger-tools, too. Thanks for the suggestion!